include/dlaf/matrix/panel.h

```diff
@@ -320,14 +373,21 @@ struct Panel : public Panel<axis, const T, device> {
   ///
   /// @pre index must point to a tile which is internally managed by the panel
   hpx::future<TileType> operator()(const LocalTileIndex& index) {
+    // Note assertion on index done by linearIndex method.
```

Comment:
Is it better to just put an assertion here as well? The comment and an assertion are both one line.

Reply:
I would avoid to duplicate assertions as it makes harder to maintain the code.
Note: I fixed the assertion in `linearIndex` as it was not complete => 1 change against 4 changes (if duplicated).

Anyway... assertion was two lines (getting idx is needed as well), and now is 3 lines...